test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ExtendedQueryTagTests.cs

```diff
@@ -24,15 +24,20 @@ public class ExtendedQueryTagTests : IClassFixture<HttpIntegrationTestFixture<St
     {
         private readonly IDicomWebClient _client;
         private const string PrivateCreatorName = "PrivateCreator1";
+
+        private const string PrereleaseV1VersionPath = "/v1.0-prerelease";
+
         public ExtendedQueryTagTests(HttpIntegrationTestFixture<Startup> fixture)
         {
             EnsureArg.IsNotNull(fixture, nameof(fixture));
             EnsureArg.IsNotNull(fixture.Client, nameof(fixture.Client));
             _client = fixture.Client;
         }
 
-        [Fact]
-        public async Task GivenValidExtendedQueryTags_WhenGoThroughEndToEndScenario_ThenShouldSucceed()
+        [Theory]
+        [InlineData("")]
+        [InlineData(PrereleaseV1VersionPath)]
```

Comment:
Instead of inline data, perhaps we should store these centrally for use across all of the tests with `[ClassData]`?

Reply:
Or are the tests varied enough we cannot really do it? Is there a way to minimize the effort of testing each new version?

Reply:
Thanks, I'm new to xUnit so just took a look at ClassData & MemberData. Let me know what you think of the updates - it should be much easier to update to new versions now, but I had to hide some of the other inline data.